Changelog.md

```diff
@@ -1,9 +1,26 @@
 # Motoko compiler changelog
 
+## 0.14.2 (FUTURE)
+
 * motoko (`moc`)
 
-  * bugfix: `mo-doc` will now generate documentation for `actor`s and `actor class`es (#4905).
+  * Added support for sending `cycles` and setting a `timeout` in parentheticals.
+    This is an **experimental feature**, with new syntax, and now also allowing best-effort
+    message sends. The legacy call `Cycles.add<system>` is still supported, but the
+    two approaches cannot be mixed (#4608).
```

Comment:
See comment in desugar.ml and how to make them coexist.

Reply:
Why would coexistence be important? If the user attaches a parenthetical, why not go all the way?

Reply:
I suppose its weird if someone specifies just a timout but that clears any cycles they added using Cycles.add. Probably rare but we don't give them any compiler warnings about mixing both, so maybe it would just be better to make it work.

Reply:
Hmm, Okay, 0189496f513ed7786df3eb75c0bc4cf54cd961f5 changed the default. Test fa768703d3f0b48d8f215e8d25aa01d6dc8f2f84 shows that `Cycles.add` now indeed punches the parenthetical.